### PR TITLE
Add content to high-level overview

### PR DESCRIPTION
**Pull request by larsgsvensson**
Hi @RubenVerborgh, I just noticed that we didn't have any content in the high-level overview section so I made a first try. Please do any changes you find appropriate!

### CODE REVIEW

File: I-D-Profile-Negotiation.xml [hunk 1]
```diff
@@ -160,6 +160,26 @@
 				</t>
 			</section>
 			<section title="High-level overview of solution">
+				<t>The proposed solution introduces two new http header fields: Accept-Profile and Content-Profile.
+					The typical use of those fields is for an agent	to include an Accept-Profile header field in an http request
+					to indicate to the origin server which profile or profiles the returned payload should conform to
+					in order for the agent to be able to process it correctly.
+					An agent can specify several profiles and use quality indicators (q-values) to indicate preferences.</t>
+				<t>An origin that supports content negotiation by profile
+				  will then include one or more Content-Profile header fields in its response
```
[the remark this answers is not in the document]
Comment (larsgsvensson) [reply I-D-Profile-Negotiation.xml]: > "on or more header fields" might be confusing?

How so? The only reason I can think of is that we haven't been explicit that having one "Content-Profile" header field with several profile URIs is equivalent to having several "Content-Profile" header fields with one profile URI each. Is that what you mean?

[the remark this answers is not in the document]
Comment (RubenVerborgh) [reply I-D-Profile-Negotiation.xml]: Indeed, perhaps "will then list one or more profiles in the "Content-Profile" header field in its response".

[the remark this answers is not in the document]
Comment (larsgsvensson) [reply I-D-Profile-Negotiation.xml]: OK

File: I-D-Profile-Negotiation.xml [hunk 2]
```diff
@@ -160,6 +160,26 @@
 				</t>
 			</section>
 			<section title="High-level overview of solution">
+				<t>The proposed solution introduces two new http header fields: Accept-Profile and Content-Profile.
```
[the remark this answers is not in the document]
Comment (larsgsvensson) [reply I-D-Profile-Negotiation.xml]: OK

File: I-D-Profile-Negotiation.xml [hunk 3]
```diff
@@ -160,6 +160,26 @@
...
+					in order for the agent to be able to process it correctly.
+					An agent can specify several profiles and use quality indicators (q-values) to indicate preferences.</t>
+				<t>An origin that supports content negotiation by profile
+				  will then include one or more Content-Profile header fields in its response
+				  specifying which profile or profiles the returned payload conforms to.
+					The specified list of profiles may not be exhaustive
+					since the returned content can conform to profiles that the origin server is not aware of.
+					If there are multiple representations of the same resource available
+					and those conform to different profiles,
+					an origin server can use a Link header field together with the profile attribute
```
[the remark this answers is not in the document]
Comment (larsgsvensson) [reply I-D-Profile-Negotiation.xml]: OK

File: I-D-Profile-Negotiation.xml [hunk 4]
```diff
@@ -160,6 +160,26 @@
...
+					to point an agent to those other representations.</t>
+				<t>Similarily, an agent sending information to an origins server,
+					e. g. by using PUT or POST, can also include a Content-Profile header field in the http request
+					to state which profile or profiles the request payload conforms to.
+					If the targeted origin server cannot process content conforming to the specified profile
```
[the remark this answers is not in the document]
Comment (larsgsvensson) [reply I-D-Profile-Negotiation.xml]: > And instead follow PUT/POST by 415 if profile not supported?

Ah, I wasn't aware of [RFC 7694 §3](https://tools.ietf.org/html/rfc7694#section-3). That's indeed very similar to our client-initiated pattern! I'll re-write this part.

File: I-D-Profile-Negotiation.xml [hunk 5]
```diff
@@ -160,6 +160,26 @@
...
+					it can reply with status code 406 Not Acceptable 
+					and include an Accept-Profile header field stating which profiles the origin server can handle.</t>
```
[the remark this answers is not in the document]
Comment (larsgsvensson) [reply I-D-Profile-Negotiation.xml]: OK

**Pull request merged**